src/lib/liquidity-curve.js

```diff
@@ -204,6 +204,7 @@ function intersectLineSegments (line0, line1) {
 
 class InvalidLiquidityCurveError extends Error {
   constructor (message) {
+    // message = message + ' points:' + this.points
```

Comment:
Referencing `this.points` doesn't really make sense.. `InvalidLiquidityCurveError` doesn't have a `points` property. Try adding `points` as another parameter. Be sure to `JSON.stringify` it as part of the error message so that its readable.